zerver/webhooks/honeybadger/doc.md

```diff
@@ -0,0 +1,25 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Honeybadger will use the `honeybadger` stream by default if no stream is given
+in the URL. Make sure the selected stream exists in Zulip.
+
+**Follow these steps:**
+
+1. Go to `Alerts & Integrations` page of your `Honeybadger` Console.
```

Comment:
This step was a bit hard to follow for me. Something like "Go to the `Alerts & Integrations` page in the project settings`" would be better.